app/views/pages/index.html.haml

```diff
@@ -75,7 +75,7 @@
                         = track_icon(track)
                     .font-semibold.text-textColor7.ml-16.whitespace-nowrap +#{sample[:num_tracks]} more
 
-        = image_tag "screenshots/landing-page-exercise.jpg", alt: "", role: :presentation, category: :graphics, class: "main-graphic lg:max-w-1-2 lg:ml-80 mt-32 lg:mt-96 shadow-lgZ1", width: 668, height: 611
+        = image_tag "screenshots/landing-page-exercise.jpg", alt: "", role: :presentation, category: :graphics, class: "main-graphic lg:max-w-1-2 lg:ml-80 mt-32 lg:mt-96 shadow-lgZ1", width: '100%', height: 611
```

Comment:
I believe you should leave the `width` in still. The browser will use the width/height to calculate the aspect ratio, then respect the CSS to render at the correct size for the user's viewport.

I've verified this works and will push :)